config/state.ts

```diff
@@ -46,8 +84,13 @@ function getCurrentState(): CLIState {
 export function getStateValue<K extends keyof CLIState>(key: K): CLIState[K] {
   ensureCLIDirectory();
 
-  const state = getCurrentState();
-  return state[key];
+  try {
```

Comment:
I'm not sure it makes sense to catch all errors from `getCurrentState`. For example, I think for JSON parsing errors or incorrectly typed state it might just make sense to throw. You're only going to get into that state if someone's editing the file, and if they are, probably best to alert them that it's broken rather than proceed as normal with the default values